.github/workflows/reviewdog.yml

```diff
@@ -15,6 +15,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # In order to take diff, commits other than the target are also required
```

Comment:
Can we remove these changes and use GitHub API by default? (See the other comment)

Reply:
I revert it: https://github.com/reviewdog/reviewdog/compare/d72ab6bf3df88c3244619648365d7114a7a1af3d...0fc18a2d37e4ccb5397c924bfdb26835f6270af2